static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -519,7 +523,9 @@ define([
         },
         allAdsRendered = function (slotId) {
             if (slots[slotId] && !slots[slotId].isRendered) {
+                slots[slotId].isLoading = false;
                 slots[slotId].isRendered = true;
+                slots[slotId].slot = null;
```

Comment:
If the node is removed from the page, the memory will be GCed on page navigation. I would personally keep this PR focused on the production issue.